### PR TITLE
Bach test athena bq improvements

### PR DESCRIPTION
**Pull request by thijs-obj**
Two changes:
* Make some tests work with Athena/BigQuery:
  * `test_positional_slicing()`
  * `test_series_unstack()`
* Created issues for non-working tests, and linked those in the `skip_*_todo` marks

### CODE REVIEW

File: bach/tests/functional/bach/test_series.py [hunk 1]
```diff
@@ -62,21 +63,30 @@ def __getitem__(self, key):
...
+        all_expected_data.extend(expected_data)
+
+    df = all_dfs[0].append(all_dfs[1:])
+    df = df.reset_index(drop=True)
+    df = df.sort_values(by=['slice', 'inhabitants'])
+    assert_equals_data(
+        df,
+        expected_columns=['inhabitants', 'slice'],
+        expected_data=all_expected_data,
+    )
```
Comment (KathiaBarahona) [inline bach/tests/functional/bach/test_series.py:89]: :+1:

File: bach/tests/functional/bach/test_series.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 """
 Copyright 2021 Objectiv B.V.
 """
+from typing import List
```
Comment (KathiaBarahona) [inline bach/tests/functional/bach/test_series.py:4]: out of the scope of this PR, but could you change the import of `assert_equals_data` from `bach.testing` instead?

**Review by KathiaBarahona — APPROVED** [review]

File: bach/tests/functional/bach/test_series.py (hunk 2 above)
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_series.py:4]: same for the other updated test files

Comment (thijs-obj) [reply bach/tests/functional/bach/test_series.py:4]: Fixed all imports in one go here: https://github.com/objectiv/objectiv-analytics/pull/1383

**Pull request merged**